src/data/users/user.db.datasource.ts

```diff
@@ -0,0 +1,31 @@
+import { prismaQuery } from '@core/db/db.js';
+import type { UserData } from '@models/users.model.js';
+import type { PrismaClient, User } from '@prisma/client';
+
+export const createUser = async (data: UserData): Promise<User> => {
```

Comment:
```suggestion
export async function create(data: UserData): Promise<User> => {
```

A parte de converter pra function é nit, só sugeri porque é assim no template.